### PR TITLE
Fix url service to handle projects that are not in the root web directory

### PR DESCRIPTION
**Pull request by dschissler**

### CODE REVIEW

File: templates/project/micro/services.php [hunk 1]
```diff
@@ -25,10 +25,14 @@
  * The URL component is used to generate all kind of urls in the application
  */
 $di->setShared('url', function () {
-    $config = $this->getConfig();
+    $scriptName = $this->getRequest()
+        ->getServer('SCRIPT_NAME');
+
+    $baseUri = preg_replace('/public\/index.php$/', '', $scriptName);
```
Comment (sergeyklay) [inline templates/project/micro/services.php]: The 
```nginx
root /usr/share/nginx/html; 
```

break it

File: templates/project/modules/config.ini [hunk 2]
```diff
@@ -11,4 +11,3 @@ controllersDir = ../apps/frontend/controllers/
 modelsDir      = ../apps/frontend/models/
 migrationsDir  = ../apps/migrations/
 viewsDir       = ../apps/frontend/views/
-baseUri        = /@@name@@/
```
Comment (sergeyklay) [inline templates/project/modules/config.ini]: Do you suggest disallow user to configure this and just hardcode this rule?

Comment (dschissler) [inline templates/project/modules/config.ini]: Yes.  I think that if anything it should just be allowing the user to configure a prefix or postfix on something.

File: templates/project/micro/services.php (hunk 1 above)
Comment (dschissler) [inline templates/project/micro/services.php]: I'm not currently able to test with nginx due to another custom environment.

File: templates/project/modules/config.ini (hunk 2 above)
Comment (sergeyklay) [inline templates/project/modules/config.ini]: 10 out of 10 frameworks allows user to do it myself. Can you name at least one reason, why hardcoded option it is OK

File: templates/project/micro/services.php (hunk 1 above)
Comment (sergeyklay) [inline templates/project/micro/services.php]: Devtools is not desogned for your environment

File: templates/project/modules/config.ini (hunk 2 above)
Comment (dschissler) [inline templates/project/modules/config.ini]: 1) I said so.

Ok so I'll dump this and give it another go in some days.

**Pull request closed without merging**